### ISSUE TITLE
Just use random number or uuid in download csv 

### ISSUE THREAD

**Issue by kljensen** [kljensen/yale-class-chat#108]
https://github.com/kljensen/yale-class-chat/blob/510f78a553ecbf2190073c9e928e535039ff812b/app/lib/app/submissions.ex#L1295

to ensure unique filename

**Comment by nperanzi** [comment]
@kljensen should we still do this now that we don't interact with the filesystem anymore?

**Comment by kljensen** [comment]
nope, no need, unless maybe if you need something similar for the file name
that the user downloads? in that case, timestamp likely best
